### PR TITLE
Full Kolmogorov PDE solution solver

### PR DESCRIPTION
**Pull request by ashutosh-b-b**
Started working on #184 .

### CODE REVIEW

**Comment by ashutosh-b-b** [comment]
@ChrisRackauckas is [this](https://github.com/SciML/NeuralPDE.jl/blob/fdf4f2ee6062c85017efe35d31a703318931b527/src/NeuralPDE.jl#L94) a correct way of taking input for different domains?

File: test/runtests.jl [hunk 1]
```diff
@@ -31,7 +31,9 @@ const is_CI = haskey(ENV,"CI")
   if GROUP == "All" || GROUP == "NNRODE"
         @time @safetestset "NNRODE" begin include("NNRODE_tests.jl") end
   end
-
+  if GROUP == "All" || GROUP == "NNPKOLMOGOROV"
+        @time @safetestset "NNParamKolmogorov" begin include("NNParamKolmogorov_tests.jl") end
```
Comment (ChrisRackauckas) [inline test/runtests.jl:35]: How long does this test set take? I don't think it needs to be separated from NNRODE. But if it does, it needs to be added to the Github CI action.

**Comment by ChrisRackauckas** [comment]
Looks good but needs documentation.

File: test/runtests.jl (hunk 1 above)
Comment (ashutosh-b-b) [reply test/runtests.jl:35]: Just in the order of minutes I guess. Yeah we can add it with NNRODE. I ll add this one and check,

**Comment by ChrisRackauckas** [comment]
Still needs docs. Please follow up.

**Pull request merged**